client/db/bolt/upgrades.go

```diff
@@ -0,0 +1,122 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package bolt
+
+import (
+	"fmt"
+
+	"decred.org/dcrdex/dex/encode"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	initialVersion = 0
+
+	// versionedDBVersion is the second version of the database. It versions
+	// the database by persisting the version.
```

Comment:
~~`It versions the database by persisting the version.`~~

Maybe something like `The only change in DB version 1 is that the DB now stores it's version number.`